### DISCUSSION TITLE
How to link comments to specific record labeled and extract comments?

### DISCUSSION THREAD

**Discussion by kissanca** [doccano/doccano#1858] [Q&A]
Hi,
Thank you for this nice tool for human annotation.
As far as I can see, the comments are stored in the dedicated section but are not directly linked to the specific record/metadata. 
Is there a way to export the comments together with the labels? 

Thank you,
Anca N.

**Comment by david-engelmann** [comment]
I wrote a small script that demonstrates how you could get the sample and comments for samples with comments

```
doccano_client.login(username=doccano_username,
        password=doccano_password)

examples = list(doccano_client.list_examples(project_id=1))

results = []
for example in examples:
    if example.comment_count > 0:
        comments = doccano_client.list_comments(project_id=1,
                example_id=example.id)
        results.append((example,list(comments)))
```

Hope this helps!
